plugins/com.gwtplugins.gwt.eclipse.core/plugin.xml

```diff
@@ -706,6 +706,11 @@
             class="com.google.gwt.eclipse.core.properties.ui.GWTProjectPropertyPage"
             id="com.gwtplugins.gwt.eclipse.core.properties.ui.gwtProjectPropertyPage"
             name="General Settings">
+         <enabledWhen>
```

Comment:
Do you think if com.gwtplugins.gdt.eclipse.core.isGpeSelected was changed to enabledWhen IJavaProject it should work like you expected?